src/views/Register.vue

```diff
@@ -45,10 +46,18 @@ export default {
       email: "",
       password: "",
       username: "",
+      passwordConfirm: "",
+      rules: {
+        samePass: v => v === this.password || 'Passwords do not match'
+      }
     };
   },
   methods: {
     register() {
+      if (this.password != this.passwordConfirm){
+        return
```

Comment:
Should we return an error alert for the user?

Reply:
We should! Added